articles/proofs-arguments-and-zero-knowledge/chapter_12.md

```diff
@@ -0,0 +1,289 @@
+---
+Title: "Σ-Protocols and Commitments from Hardness of Discrete Logarithm"
+Status: Done
+Level: "3"
+---
+
+# Chapter 12: $\Sigma$-Protocols and Commitments from Hardness of Discrete Logarithm
+
+## 12.1 Cryptographic Background
+
+### 12.1.1 A Brief Introduction to Groups
+
+This is the definition of [group](../../terms/group.md).
+
+### 12.1.2 The Discrete Logarithm Problem and Background on Elliptic Curves
+
+#### 12.1.2.1 Discrete Log Problem
+
+You should read the definition of Discrete Logarithm [here](../../terms/discrete_logarithm.md)
+
+#### 12.1.2.2 Elliptic Curve Groups
+
+You should take a look at Elliptic curve group [here](../../terms/elliptic_curve_group.md).
+
+## 12.2 Schnorr’s $\Sigma$-Protocol For Knowledge of Discrete Logarithms
+
+This section covers:
```

Comment:
This should be `the purpose of this sum protocol` or smth like that

Reply:
![Screenshot 2024-03-26 at 10 32 17](https://github.com/sota-zk-lab/zkp-documents/assets/33181397/412a0b34-a178-4c66-806c-c266bc84d11d)
![Screenshot 2024-03-26 at 10 32 27](https://github.com/sota-zk-lab/zkp-documents/assets/33181397/ff4a9acc-418a-4cf1-9b66-11aec8a2dcd6)